report-flat/src/App.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <div id="app">
+    <ReportOptions :options="options"/>
+    <div class="suite">
+      <hr/>
+      <img id="logo" src="./assets/logo.png">
+      Company: {{suite.company}} Project: {{suite.project}} Name: {{suite.name}} CorrelationID:
+      {{suite.correlationId}}
+      <hr/>
+      <div class="test" v-for="test in suite.tests">
```

Comment:
Do you use some formatting for HTML? It looks inconsitent, Webstorm is capable of formatting Vue templates, if you are using some sort of opensource editor probably there is plugin for that